packages/admin-next/dashboard/src/components/table/data-table/data-table-filter/data-table-filter.tsx

```diff
@@ -41,10 +41,15 @@ export type Filter = {
 
 type DataTableFilterProps = {
   filters: Filter[]
+  readonly?: boolean
   prefix?: string
 }
 
-export const DataTableFilter = ({ filters, prefix }: DataTableFilterProps) => {
+export const DataTableFilter = ({
+  filters,
+  readonly,
+  prefix,
+}: DataTableFilterProps) => {
   const { t } = useTranslation()
   const [searchParams] = useSearchParams()
   const [open, setOpen] = useState(false)
```

Comment:
Question: Can't we rely solely on the query parameters for the `activeFilters` state? I feel like it should be possible, and we can remove some code around maintaining their state (so basically we have getInitialFilters be the state, and that gets memoized with `location.search` passed to it)